packages/data-stores/src/plans/types.ts

```diff
@@ -50,70 +50,32 @@ export interface PlanProduct {
 }
 
 /**
- * types of an item from https://public-api.wordpress.com/rest/v1.5/plans response
- * can be super useful later
+ * Item returned from https://public-api.wordpress.com/rest/v1.5/plans response
+ * Only the properties that are actually used in the store are typed
  */
 export interface PricedAPIPlan {
 	product_id: number;
 	product_name: string;
-	meta: Record< string, unknown >;
-	prices: {
-		AUD: number;
-		BRL: number;
-		CAD: number;
-		CHF: number;
-		CZK: number;
-		DKK: number;
-		EUR: number;
-		GBP: number;
-		HKD: number;
-		HUF: number;
-		IDR: number;
-		ILS: number;
-		INR: number;
-		JPY: number;
-		MXN: number;
-		NOK: number;
-		NZD: number;
-		PHP: number;
-		PLN: number;
-		RUB: number;
-		SEK: number;
-		SGD: number;
-		THB: number;
-		TWD: number;
-		USD: number;
-		TRY: number;
-	};
-	bundle_product_ids: {
-		0: number;
-		1: number;
-		2: number;
-		3: number;
-		4: number;
-		5: number;
-		6: number;
-	};
-	path_slug: PlanPath;
+	path_slug?: PlanPath;
 	product_slug: StorePlanSlug;
-	description: string;
-	cost: number;
-	bill_period: 31 | 365;
-	product_type: string;
-	available: string;
-	multi: number;
-	bd_slug: string;
-	bd_variation_slug: string;
-	outer_slug: string;
-	extra: string;
-	capability: string;
-	product_name_short: string;
-	bill_period_label: string;
-	price: string;
-	formatted_price: string;
+	bill_period: -1 | 31 | 365;
 	raw_price: number;
-	tagline: Record< string, unknown >;
 	currency_code: string;
+	[ key: string ]: unknown;
```

Comment:
Why do we want to include "unknown" props in our types (the same goes for `APIPlanDetail` & `DetailsAPIResponse`)?

Reply:
As per our conversation, I moved the "unknown" types declarations in the mock files, so that the rest of our code can rely of stricter type checking